frontend/src/metabase/lib/redux/hooks.unit.spec.tsx

```diff
@@ -0,0 +1,152 @@
+import React, { useEffect } from "react";
+import type { AnyAction } from "redux";
+import type { State } from "metabase-types/store";
+import type { User } from "metabase-types/api";
+
+import { renderWithProviders, screen } from "__support__/ui";
+import { useDispatch, useSelector } from "./hooks";
+import type { AppDispatch } from "./hooks";
+
+const DEFAULT_USER: User = {
```

Comment:
You can use `createMockUser` to not specify not used properties

Reply:
@ranquild Thanks, just updated it